cantools/db/message.py

```diff
@@ -108,14 +117,20 @@ def _create_message_encode_decode_formats(signals):
     for signal in signals[::-1]:
         if signal.byte_order == 'big_endian':
             continue
+    
+        if signal.is_float:
+            signal_type = 'f'
+        elif signal.is_signed:
+            signal_type = 's'
+        else:
+            signal_type = 'u'
```

Comment:
I suggest creating a function returning the format string type. Define the function within the scope of `_create_message_encode_decode_formats()`.

Somthing like:

``` python
def get_format_string_type(signal):
    if signal.is_float:
        return 'f'
    elif signal.is_signed:
        return 's'
    else:
        return 'u'
```

Reply:
Yes, I wanted to do this initially. But knowing not everybody like nested functions I did without it just to be sure ^^.